Makefile

```diff
@@ -55,14 +55,16 @@ glideup:
 build: binary-build
 	@$(call check-static-binary,deis)
 
+# This is supposed to be run within a docker container
 build-latest:
-	${DEV_ENV_CMD} gox -verbose -parallel=3 ${GO_LDFLAGS} -os="${BUILD_OS}" -arch="${BUILD_ARCH}" -output="$(DIST_DIR)/deis-latest-{{.OS}}-{{.Arch}}" .
+	gox -verbose -parallel=3 ${GO_LDFLAGS} -os="${BUILD_OS}" -arch="${BUILD_ARCH}" -output="$(DIST_DIR)/deis-latest-{{.OS}}-{{.Arch}}" .
```

Comment:
I'll trust your judgement on it. I didn't write this code, but that sounds reasonable.